src/ark/phenotyping/cluster_helpers.py

```diff
@@ -118,7 +120,7 @@ class PixelSOMCluster(PixieSOMCluster):
     def __init__(self, pixel_subset_folder: pathlib.Path, norm_vals_path: pathlib.Path,
                  weights_path: pathlib.Path, fovs: List[str], columns: List[str],
                  num_passes: int = 1, xdim: int = 10, ydim: int = 10,
-                 lr_start: float = 0.05, lr_end: float = 0.01):
+                 lr_start: float = 0.05, lr_end: float = 0.01, seed=42):
```

Comment:
I think `seed` can be added to the abstracted PixieSOMCluster since it's used for both pixel and cell objects.